src/lib/bmp/bmp.h

```diff
@@ -0,0 +1,53 @@
+/**
+  * @file bmp.h
+  * @brief lib for read and draw BMP
+  * @author Filipp Chubukov
+  * @version
+  * @date 03.07.2019
+*/
+
+#ifndef LIB_BMP_H_
+#define LIB_BMP_H_
+
+/**
+  * @brief check BMP_head
+  *
+  * @param fd File descriptor
+  *
+  * @return -1 if not BMP and 0 if BMP
+*/
+extern int check_bmp(int fd);
+
+/**
+  * @brief fill struct bmp_header
+  *
+  * @param fd File descriptor
+  * @param bmp_header - buffer for bmp_header
+  *
+  * @return -1 if error and 0 if successed
+*/
+extern int get_head_bmp(int fd, bmp_header *bmp_head);
```

Comment:
Why this function is global?

Reply:
I think that the user may want to get only information about bmp